paparazzi/paparazzi-gradle-plugin/src/test/projects/view-with-lifecycle-usages/src/test/java/app/cash/paparazzi/plugin/test/LifecycleUsageTest.kt

```diff
@@ -0,0 +1,38 @@
+package app.cash.paparazzi.plugin.test
+
+import android.view.View
+import androidx.activity.findViewTreeOnBackPressedDispatcherOwner
+import androidx.activity.OnBackPressedCallback
+import androidx.activity.OnBackPressedDispatcherOwner
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+class LifecycleUsageTest {
+  @get:Rule
+  val paparazzi = Paparazzi()
+
+  @Test fun OnBackPressedDispatcherOwner() {
+    val view = View(paparazzi.context)
+    view.doOnAttach {
```

Comment:
This test has an implicit expectation that `onAttachedToWindow()` always gets called in paparazzi tests. Is that good enough? Otherwise I can manually assert that a `OnBackPressedDispatcherOwner` owner was found after the `paparazzi.snapshot()` call.

Reply:
that's good enough!